Mathlib/Algebra/Order/Monoid/Unbundled/WithTop.lean

```diff
@@ -372,9 +372,7 @@ instance existsAddOfLE [LE α] [Add α] [ExistsAddOfLE α] : ExistsAddOfLE (With
 theorem zero_lt_top [Zero α] [LT α] : (0 : WithTop α) < ⊤ :=
```

Comment:
Probably we should deprecate this one to match, and add `top_pos``

Reply:
`zero_lt_top` seems to be a lot more popular, but I agree that we might as well do this now.